test/extensions/stats_sinks/hystrix/hystrix_test.cc

```diff
@@ -98,6 +98,24 @@ class ClusterTestInfo {
   NiceMock<Stats::MockCounter> retry_4xx_counter_;
 };
 
+class HistogramWrapper {
+public:
+  HistogramWrapper() { histogram = hist_alloc(); }
+
+  ~HistogramWrapper() { hist_free(histogram); }
+
+  const histogram_t* getHistogram() { return histogram; }
+
+  void initHistogram(const std::vector<uint64_t>& values) {
```

Comment:
Why not in constructor?

Reply:
Just since it can be called several times to set different values (IIUC). Although we don't do that in code. So maybe I'll change the name to `setHistogramValues`